content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Troubleshooting Guide
+description: This article helps common troubleshooting issues with XSOAR integrations
+---
+
+# Troubleshooting Guide
+
+##Connection issues troubleshooting guide:
+### When a customer has any connection errors try the following steps:
+1. Check and uncheck the “insecure” checkbox
+2. Check and uncheck the “proxy” checkbox
+3. Try to isolate the problem, is it the integration instance server or the client?
+
+    a. In case you have access from your local computer to the integration instance server, Try to generate the same request using curl or postman from your local computer (or customer’s computer)- if it fails: the integration instance server is probably down, could be instance issue.
+    
+    b. Try to do some curl from the XSOAR machine, for example: 
+    ```
+   curl https://httpbin.org/status/200  -vk
+   ```
+    and see if the returning status is 200
+
+    c. Try to run some curl request via a docker container using 
+    ```
+   docker run -it demisto/netutils:1.0.0.6138 curl https://httpbin.org/status/200  -vk
+   ```
+   
+    - If section c. fails and you suspect this is a proxy issue, try to run curl without proxy using 
+    ```
+   docker run -it demisto/netutils:1.0.0.6138 curl https://httpbin.org/status/200  -vk --noproxy "*"
+   ```
+   
+    - If section b. worked and section c. failed: this could be a docker networking issue, see the following [link](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/5-5/cortex-xsoar-admin/docker/docker-hardening-guide/troubleshoot-docker-networking-issues) for guidance.
```

Comment:
link is working:
![image](https://user-images.githubusercontent.com/30797606/84145432-88641a00-aa62-11ea-81fb-169ccdd44a46.png)

Reply:
Is working? what's wrong with that?
Can you elaborate what's wrong here?

Reply:
sorry, I meant that the markdown of the link isn't working it doesn't hide the URL and it is not clickable.